### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by bealecs**
I have committed the week 3 issue document inside of the week 3 folder. Check out the readme for instructions and link to 
 Netlify preview

### CODE REVIEW

File: ThirdReadMe.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# **Clif Codes**
+
+### *This is serving as my changes to the MarkDown with VS code*
+
+#### I added a new branch named 'secondary brach' and I will be committing this file to the secondary-branch and then opening the pull request to the main branch from there
```
Comment (axecopfire) [inline ThirdReadMe.md]: This is a really long title

File: ThirdReadMe.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# **Clif Codes**
```
Comment (axecopfire) [inline ThirdReadMe.md]: You don't need to Bold headings

File: Week3/index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+    <!-- Header Section -->
+    <header>
+        <navbar style="display: flex; flex-direction: row; justify-content: space-evenly;">
```
Comment (axecopfire) [inline Week3/index.html]: This is supposed to be week3 html, there shouldn't be any CSS. Also, when you do end up using CSS. Please don't use inline CSS. Default to using a CSS file

File: Week3/index.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+            <!-- Unorderd list Section -->
+            <ul style="display: flex; flex-direction: row; justify-content: space-between; margin: auto; width: 80%;">
+                <li style="text-decoration: underline; list-style: none; ">Nav Item 1</li>
+                <li style="text-decoration: underline; list-style: none; ">Nav Item 2</li>
+                <li style="text-decoration: underline; list-style: none; ">Nav Item 3</li>
+            </ul>
+        </navbar>
+    </header>
+
+    <body>
```
Comment (axecopfire) [inline Week3/index.html]: All content should be nested inside the `<body>`. Only the `<head>` should be outside.

File: Week3/index.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                <td style="margin: auto;border: 1px solid black">Table Data Column</td>
+            </tr>
+            <tr style="margin: auto;border: 1px solid black">
+                <td style="margin: auto;border: 1px solid black">Table Data Column</td>
+                <td style="margin: auto;border: 1px solid black">Table Data Column</td>
+                <td style="margin: auto;border: 1px solid black">Table Data Column</td>
+            </tr>
+
+            <tr style="margin: auto;border: 1px solid black">
+                <th style="text-align: center;">Footer</th>
```
Comment (axecopfire) [inline Week3/index.html]: `<th>` stands for header, seems strange to use as a footer.

File: Week3/index.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+            </tr>
+        </table>
+    </body> 
+
+    <!-- Footer Section -->
+    
+    <footer>
```
Comment (axecopfire) [inline Week3/index.html]: `<footer>` needs to be inside `<body>`.

**Review by axecopfire — CHANGES_REQUESTED** [review]
Check this out for an example with some code that uses `<header>` and `<footer>`

https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#active_learning_exploring_the_code_for_our_example

File: Week3/index.html (hunk 3 above)
Comment (bealecs) [reply Week3/index.html]: Noted, I will fix that now

File: Week3/index.html (hunk 4 above)
Comment (bealecs) [reply Week3/index.html]: Roger that

File: Week3/index.html (hunk 5 above)
Comment (bealecs) [reply Week3/index.html]: Just now realized there is a <tfoot> tag.... I have always used <th> LOL thanks, will also fix now

File: Week3/index.html (hunk 6 above)
Comment (bealecs) [reply Week3/index.html]: Roger

**Review by axecopfire — APPROVED** [review]
Great job and thanks for working through these comments.

**Pull request merged**